dbclient/dbclient.py

```diff
@@ -6,6 +6,7 @@
 from dbclient import parser
 import time
 import requests.packages.urllib3
+import logging
```

Comment:
Can we add common error logging in dbclient which shared by all clients?

It'll release our burden for adding error logging in each of the clients. We can only add extra logging for important errors.

Reply:
If we want to add failures in dbclient, then we need to compliment all calls with action_type, object_type in order add failures to the right file. After auditing the code, it seemed like clients are printing custom failed logs from many places. In these cases, adding the error logging in dbclient will push duplicate error logs for the same failure and there is no easy way to avoid it.  Therefore I kept it as client responsibility.

I can add a util function which can be called to log error by action_type, object_type across all clients.

Reply:
I mean we can log all errors into one file as the fallback just in case that we forget to add error logs for certain object types.

Reply:
done